types/Keyguard.d.ts

```diff
@@ -121,7 +223,12 @@ type ConstructSwap<T extends KeyguardRequest.SignSwapRequestCommon> = Transform<
                 keyPath: string,
             },
             output: KeyguardRequest.BitcoinTransactionChangeOutput,
-        } | {
+        } | (
+            { type: 'USDC' } & Omit<KeyguardRequest.PolygonTransactionInfo, 'approval'> & {
```

Comment:
```suggestion
            { type: 'USDC' } & Omit<KeyguardRequest.PolygonTransactionInfo, 'approval' | 'amount'> & {
```
Should the optional `amount` be omitted as it gets added as required property, or does it make no difference in the resulting type?